help_to_heat/templates/frontdoor/epc.html

```diff
@@ -46,6 +46,12 @@ <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title
         <p class="govuk-body">{{ epc_rating.upper() }}</p>
       {% endif %}
       
+      <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-1">{{_("Date of issue")}}</p>
+
+      {% if gds_epc_date %}
```

Comment:
Do we always expect to have this?

Reply:
Yes, we should be able to always expect this. I've removed this check now